From: Thomas Gleixner <tglx@linutronix.de>
Date: Mon, 4 Jul 2016 15:32:25 +0200
Subject: genirq/msi: Fix broken debug output
Patch-Mainline: v4.8
Git-commit: 4364e1a29be16b2783c0bcbc263f61236af64281
References: bsc#998391

virq is not required to be the same for all msi descs. Use the base irq number
from the desc in the debug printk.

Reported-by: Ingo Molnar <mingo@kernel.org>
Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Acked-by: Hannes Reinecke <hare@suse.com>
---
 kernel/irq/msi.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/kernel/irq/msi.c b/kernel/irq/msi.c
index 4769781..c0f385e 100644
--- a/kernel/irq/msi.c
+++ b/kernel/irq/msi.c
@@ -300,6 +300,7 @@ int msi_domain_alloc_irqs(struct irq_domain *domain, struct device *dev,
 		ops->msi_finish(&arg, 0);
 
 	for_each_msi_entry(desc, dev) {
+		virq = desc->irq;
 		if (desc->nvec_used == 1)
 			dev_dbg(dev, "irq %d for MSI\n", virq);
 		else
-- 
1.8.5.6

